docs/index.md

```diff
@@ -1,477 +0,0 @@
-instance which implements the [DeepMind Environment API][dm_env]. Mava currently supports multi-agent environment loops and environment wrappers for the following environments and environment suites:
-
-* [PettingZoo][pettingzoo]
-* [SMAC][smac]
-* [Flatland][flatland]
-* [2D RoboCup][robocup]
-* [OpenSpiel][openspiel]
-* [Melting pot][meltingpot]
-
-For details on how to add your own environment, see [here](https://github.com/instadeepai/Mava/blob/develop/mava/wrappers/README.md).
-
-|<img  src="docs/images/multiw_animation.gif" width="300px"/> | <img src="docs/images/sc2_animation.gif" width="200px"/>  | <img src="docs/images/flatland.gif" width="200px" />  |
-|:---:|:---:|:---:|
-|MAD4PG on PettingZoo's Multi-Walker environment. | VDN on the SMAC 3m map.| MADQN on Flatland. |
-
-|<img  src="docs/images/robocup_animation.gif" width="300px"/> |<img  src="docs/images/madqn_meltingpot_cleanup_scenario.gif" width="300px"/> |
-|:---:|:---:|
-|MAD4PG on the 2D RoboCup environment using 6 executors.| MADQN on a melting pot clean up scenario |
-
-## Tensorflow System Implementations
-
-| **Name**         | **Recurrent**      | **Continuous** | **Discrete**  | **Centralised training**  | **Multi Processing**   |
-| ------------------- | ------------------ | ------------------ | ------------------ | ------------------- | ------------------- |
-| MADQN   | ✔️ | ❌ | ✔️ | ❌ | ✔️ |
-| MADDPG  | ✔️ | ✔️ | ✔️       | ✔️  | ✔️ |
-| MAD4PG   | ✔️ | ✔️ | ✔️ | ✔️  | ✔️ |
-| MAPPO   | ❌ | ✔️ | ✔️ | ✔️ | ✔️ |
-| VDN   | ✔️ | ❌ | ✔️ | ✔️ | ✔️ |
-| QMIX   | ✔️ | ❌ | ✔️                | ✔️ | ✔️ |
-
-## Jax System Implementations
-
-| **Name**         | **Recurrent**      | **Continuous** | **Discrete**  | **Centralised training**  | **Multi Processing**   |
-| ------------------- | ------------------ | ------------------ | ------------------ | ------------------- | ------------------- |
-| IPPO   | ❌ | ❌ | ✔️ | ❌ | ✔️ |
-
-As we develop Mava further, we aim to have all systems well tested on a wide variety of environments. The team is also hard at work at expanding the systems that are implemented in Jax using the callback design approach.
-
-## Usage
-
-To get a sense of how Mava systems are used we provide the following simplified example of launching a Tensorflow based distributed MADQN system.
-
-```python
-# Mava imports
-from mava.systems.tf import madqn
-from mava.components.tf.architectures import DecentralisedPolicyActor
-from . import helpers
-
-# Launchpad imports
-import launchpad
-
-# Distributed program
-program = madqn.MADQN(
-    environment_factory=helpers.environment_factory,
-    network_factory=helpers.network_factory,
-    architecture=DecentralisedPolicyActor,
-    num_executors=2,
-).build()
-
-# Launch
-launchpad.launch(
-    program,
-    launchpad.LaunchType.LOCAL_MULTI_PROCESSING,
-)
-```
-
-We also illustrate how a Jax based IPPO system may be implemented.
-
-```python
-# Mava imports
-from mava.systems.jax import ippo
```

Comment:
Why did we remove this index page?

Reply:
Hmm, will re-add this. Basically, this is the old readme that we use in the docs now. I will add a reference to our current main, so the docs and the readme are consistent.

Reply:
@DriesSmit  I have re-added this to be the same as our readme.

Reply:
Thanks :slightly_smiling_face: